base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -472,23 +477,72 @@ where
                     request: Some(request),
                 };
                 // TODO Remove this once this bug is fixed
-                trace!(target: LOG_TARGET, "About to attempt to send query to base node");
-                self.outbound_message_service
+                trace!(
+                    target: LOG_TARGET,
+                    "Attempting UTXO query ({}) to base node",
+                    request_key
+                );
+                match self
+                    .outbound_message_service
                     .send_direct(
                         pk.clone(),
                         OutboundEncryption::None,
                         OutboundDomainMessage::new(TariMessageType::BaseNodeRequest, service_request),
                     )
-                    .await?;
+                    .await
+                {
+                    Ok(result) => match result.resolve_ok().await {
```

Comment:
We can't do this like this because of the timing implications. This can potentially block for minutes and will stop the main loop in the service task to become unresponsive.

Reply:
Reverted changes to `base_layer/wallet/src/output_manager_service/service.rs` as discussed